cli/Valet/PhpFpm.php

```diff
@@ -192,8 +253,31 @@ public function useVersion($version, $force = false)
             $this->brew->ensureInstalled($version, [], $this->taps);
         }
 
-        // Unlink the current php if there is one
+        // Delete old Valet sock files, install the new version, and, if this is a global change, unlink and link PHP
+        if ($directory && $site) {
+            $customPhpVersion = $this->site->customPhpVersion($site); // Example output: "74"
+            $this->cli->quietly('sudo rm '.VALET_HOME_PATH.'/'.$this->fpmSockName($version));
+            $this->updateConfiguration($version);
+            $this->site->installSiteConfig($site, $this->fpmSockName($version), $version);
+
+            $this->maybeStop($customPhpVersion);
+            $this->restart($version);
+            $this->nginx->restart();
+            info(sprintf('The [%s] site is now using %s.', $site, $version));
```

Comment:
What's the shape of `$version` at this point? is it something like `7.4`? If so, I'd love to make this change:

```suggestion
            info(sprintf('The site [%s] is now using PHP %s.', $site, $version));
```

Reply:
`$version` would be something like `php@7.4`. Check the output in action. 

<img width="339" alt="CleanShot 2022-02-16 at 11 26 47@2x" src="https://user-images.githubusercontent.com/13833460/154201805-5d8ea2d5-338b-4bcf-bb0e-e2dcd6549e7b.png">